airbyte-webapp/src/components/EntityTable/components/StreamStatusCell.module.scss

```diff
@@ -0,0 +1,84 @@
+@use "scss/colors";
+@use "scss/variables";
+
+.bar {
+  width: 100%;
+  max-width: 370px;
+  height: 23px;
+  border-radius: variables.$border-radius-xs;
+  overflow: hidden;
+  position: relative;
+  display: flex;
+  gap: 1px;
+
+  .filling {
+    width: 100%;
+    height: 100%;
+
+    &.onTrack {
+      background: colors.$green;
+    }
+
+    &.behind {
+      background: colors.$blue;
+    }
+
+    &.error {
+      background: colors.$red;
+    }
+
+    &.disabled {
+      background: colors.$grey;
+    }
+  }
+}
+
+$contentWidth: 75%;
```

Comment:
Instead of using a fixed width, it's better to use horizontal padding to push the content inward.